feat: criando a seção principal que contém o mapa do IFPB-CG

jpalvesl opened the pull request.

jpalvesl committed:
feat: criando a seção principal que contém o mapa do IFPB-CG

henriquepw reviewed (changes requested):
- Cria o header de acordo com a interface criada.
- Muda essa SecãoPrincipal para o index

Judenilson committed:
chore: adicionado Header, Footer e Caixa de Pesquisa

jpalvesl committed:
refactor: separando header e footer em componentes

alem de separar em componentes separa sua estilizaçao tambem

jpalvesl committed:
chore: caixa de pesquisa se contra agora na mesma div do icone de busca

a cor do icone de busca agora se altera para que não fique invisivel na caixa de pesquisa

jpalvesl committed:
style: removendo as pastas do atomic desgin e alterando local do seo

henriquepw committed:
feat: config do import-helpers no eslint

configurando corretamente o import-helpers no eslint para organizar os imports dos components

henriquepw commented:
Atualizei o `eslint` para ele poder organizar as importações da pasta `components` corretamente.

henriquepw committed:
chore: adicionando icone de pesquisa nos assets

svg do icone usado na barra de pesquisa adicionado

henriquepw commented:
Adicionei também o `.svg` do ícone correto da barra de pesquisa

henriquepw reviewed (changes requested):
- Usar o `styled-components`
- Atualizar o arquivo `themes.js` com as cores corretas
- Atualizar o `Header` de acordo com o design feito

:bulb: dica: o arquivo com o design no `figma` já da várias informações para criar o `css`, alias, se clicar em algum componente, ele já da todo o `css`, mas não use ele completo, use mais para saber os espaçamentos.

jpalvesl committed:
chore: insere o svg na seção principal e altera o background com o hover

usando o svg-inline-react é colocado o svg na seção principal e usando o mousehover para alterar o background de cada sala ou bloco do svg

Judenilson committed:
chore: adição de arquivos .json

Judenilson committed:
chore: atualização Arquivos .JSON

Alteração dos campos vazios trocando separador e palavras em maiúscula.

Judenilson committed:
chore: atualização Arquivos .JSON criação chave do array

jpalvesl committed:
chore: adicionando os mapas em formato svg

jpalvesl committed:
chore: adicionando o plugin do gatsby para o uso de svg's

remove o pacote anteriormente utilizado para usar os svg's e adiciona o plugin do gatsby para isso

jpalvesl committed:
chore: alterando cores de themes e colocando transition no hover do mapa

jpalvesl committed:
refactor: usando styled component pra criar o footer

remove a estilização presente em Layout referente ao footer e agora ela se encontra no styled component FooterSC

jpalvesl committed:
refactor: alterando icone de presquisa/usando styled component em header

jpalvesl committed:
chore: criando a caixa de pesquisa como um componente

remove as dependencias no componente HeaderSC referente a caixa de pesquisa e cria um componente
separado para isso

henriquepw commented:
No header, deixa a lupa clicável e usa um state para dizer se ele ta aberto ou fechado.


Presta atenção no modelo do Figma, os ícones ficão juntos da direita, em vez da lupa no meio, se você selecionar um componente no Figma, segura `Alt` e bota o mouse em cima de outro, ele diz a distancia entre eles.

![Screenshot from 2020-03-21 10-28-10](https://user-images.githubusercontent.com/16365204/77227393-d00f8f80-6b5e-11ea-8dcc-a3429ea4198e.png)

Tem a parte de `code` do Figma tbm, que ajuda.

![Screenshot from 2020-03-21 10-26-43](https://user-images.githubusercontent.com/16365204/77227367-a5bdd200-6b5e-11ea-86c2-4ba38754f2f1.png)

jpalvesl committed:
refactor: posicionando os itens corretamente no header

alem de posicionar os icones, agora a lupa também é clicavel e quando esse evento ocorro ou a tecla
enter é pressionada na caixa de pesquisa uam função é chamada

jpalvesl commented:
Posicionei os icones de acordo com o Wireframe, porém mesmo com a largura 0 o input continua aparecendo antes de clicar na lupa(da pra ver até pelo mouse se a pessoa for com ele a esquerda da lupa), eu até tentei usar um useState pra deixar "display: none" e depois quando eu clicasse na lupa fosse pra "display: block", mas isso faz com que não tenha animação

jpalvesl committed:
fix: fazendo com que a caixa de pesquisa realmente suma do header

jpalvesl committed:
chore: aumentando o margin-top do main e diminuindo tamanho do svg

jpalvesl committed:
feat: adicionando uma janela provisoria para visualizar a sala com hover

jpalvesl committed:
chore: fazendo com que a janela seja realmente flutuante

jpalvesl committed:
chore: ajeitando o footer pois a estilização do texto estava errada

henriquepw commented:
- Ao clicar novamente na lupa, eu não consigo fechar o a caixa de pesquisa, ao invés disso ele abre um alert:
![Alert](https://user-images.githubusercontent.com/16365204/77824164-67389200-70df-11ea-9787-4e4a5d4c0b5d.png)

-A janela flutuante ficaria melhor se fosse com opacidade 1, do jeito que ta dificulta a leitura.

jpalvesl committed:
chore: removendo divs desnecessarias e transformando searchBox em form

jpalvesl committed:
chore: transforma JSON em array de objetos e refatora pagina principal

jpalvesl committed:
chore: removendo title dos SVGs e adicionando fonte do wireframe

jpalvesl committed:
chore: adicionando conceito de responsividade no header

jpalvesl committed:
chore: alterando zIndex relacionados ao layout e mudando nome de arquivo

alterando o nome do arquivo recepcao.json por antiga-recepcao.json adicionado uma margin-bottom com
relacao a tag main do layout e alterando z-index de do main e da janela flutuante

jpalvesl committed:
chore: adicionando todos as outras paginas que ja tem o mapa svg feito
